backend/config/packages/twig.yaml

```diff
@@ -0,0 +1,6 @@
+twig:
```

Comment:
I don't think we want to use twig in this project so twig bundle is unnecessary

Reply:
We need a twig bundle to use `make` command in the future.